script/passenger-recycler.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Trivial Passenger memory monitor installed via /etc/cron.hourly/. To modify
+# configuration create /etc/foreman/passenger-recycler.rb.conf file.
+#
+
+CONFIG = '/etc/foreman/passenger-recycler.rb.conf'
```

Comment:
I would vote for a config file.. foreman_ansible and the Foreman inventory for Ansible can be configured via environment variables. The result is that it leads to confusion to users, "where do I persist this?", "which user should have the environment variable?", "should these variables be global?", etc...